src/sentry/rules/conditions/event_attribute.py

```diff
@@ -54,7 +54,7 @@ class MatchType(object):
 
 class EventAttributeForm(forms.Form):
     attribute = forms.ChoiceField((a, a) for a in ATTR_CHOICES)
-    match = forms.ChoiceField(MATCH_CHOICES.items())
+    match = forms.ChoiceField(list(MATCH_CHOICES.items()))
```

Comment:
I'm still trying to catch up with Python best practices and efficiencies. Do you mind explaining why `tuple` is more efficient than `list` here? Since there aren't many items I would have thought they were interchangeable.

Reply:
What @wedamija said is correct. The reason I suggested using a `tuple` is because it communicates the intent: `tuple` is immutable and `list` is mutable. That's all.

Some performance tests show that creating a `tuple` is slightly more expensive than a `list` but access is ~2x faster. But again, performance is irrelevant in this context.